components/mention/index.tsx

```diff
@@ -101,6 +103,16 @@ export default class Mention extends React.Component<MentionProps, MentionState>
       this.props.onBlur(ev);
     }
   }
+  focus = () => {
+    this.mentionEle._editor.focus();
+  }
+  mentionRef = ele => {
+    const { ref } = this.props;
```

Comment:
I don't understand, it seems that we [cannot read ref from props](https://github.com/facebook/react/blob/1c449cea348bc18add78c5892af332e6febd38b5/src/isomorphic/classic/element/ReactElement.js#L208).